apps/web/src/app/[locale]/layout.tsx

```diff
@@ -62,26 +10,11 @@ export default function RootLayout({
   params: { locale: string };
 }) {
   return (
-    <html suppressHydrationWarning lang="en">
-      <head>
-        <meta
-          content="Level up your typescript skills with interactive exercises"
-          name="description"
-        />
-        <link href="/favicon.svg" rel="icon" type="image/svg+xml" />
-        <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
-        <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
-        <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
-        <link rel="manifest" href="/site.webmanifest" />
-      </head>
-      <body className={`${inter.className} flex flex-col`}>
-        <Providers locale={locale}>
-          <Navigation />
-          <main className="flex-1">{children}</main>
-          <Toaster />
-        </Providers>
-        <Analytics />
-      </body>
-    </html>
+    <>
+      <Navigation />
+      <main className="flex-1">
+        <I18nProviderClient locale={locale}>{children}</I18nProviderClient>
+      </main>
```

Comment:
Shouldn't `Navigation` be wrapped inside `I18nProviderClient`?

Reply:
Really not sure. In your example all nav is in the layout.tsx and the provider is used to wrap some client components that dont have any nav? 

Or am i missing it?

I can try it tho and see what happens.

Reply:
It seems to work... is this the recommended approach? Why is the app route example different?

Reply:
thats weird that it works not being nested 😅

Reply:
confirmed Navigation needs to be inside the provider.... the reason it worked before was we weren't doing any i18n inside the nav component.  Once I added i18n the component would break if it was outside the context provider

Reply:
Yeah exactly this ^